application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCommand.java

```diff
@@ -0,0 +1,268 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.component.ButtonInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BookmarksCommand extends SlashCommandAdapter {
+
+    public static final long EXPIRE_DELAY_SECONDS = 3L * 24 * 60 * 60; // 3 Days
+
+
+    private static final MessageEmbed EMBED_NOT_IN_DM =
+            new EmbedBuilder().setTitle("This command cannot be used in a DM!")
+                .setColor(0xFF8484)
+                .build();
```

Comment:
Create a method which makes the embed for you, taking the title as an argument/

So it'll look like
```suggestion
    private static final MessageEmbed EMBED_NOT_IN_DM = generateEmbed("This command cannot be used in a DM!");
```

instead of the current code, which looks very bloating

Reply:
Your right. The final embeds are the same anyways but with different text and color. Im gonna do that real quick.

Reply:
Ok done